EIPS/eip-1153.md

```diff
@@ -33,7 +33,7 @@ Potential use cases enabled or improved by this EIP include:
 
 1. Reentrancy locks
 2. On-chain computable CREATE2 addresses: constructor arguments are read from the factory contract instead of passed as part of init code hash
-3. Single transaction [ERC-20](./eip-20.md) approvals, e.g. `#temporaryApprove(address spender, uint256 amount)`
+3. Single transaction [ERC-20](./eip-20.md)-esque approvals, e.g. `#temporaryApprove(address spender, uint256 amount)`
```

Comment:
In this usage, the EIP appears to misrepresent ERC20 approve as use case but refers to a method that is not part of ERC20.